getting_started_vagrantbox.md

```diff
@@ -0,0 +1,85 @@
+
+TODO: generate PNG
+```mermaid
+graph TD
+  subgraph local machine
+    subgraph "virtual machine (10.0.3.10)"
+        Consul
+        Nomad
+        Vault
+        vagrant["/vagrant"]
+        MinIO
+        Other[Other softwares installed]
+    end
+    localhost["localhost (127.0.0.1)"]
+    pwd["/<path-to-vagrantfile>"]
+  end
+  %% links
+  Consul-->localhost
+  Nomad-->localhost
+  Vault-->localhost
+  vagrant-->pwd
+```
+
+### Services inside box
+The virtual machine has four services running:
+
+|Name|port|
+|:--|:-:|
+|Consul|8500|
+|Vault|8200|
+|Nomad|4646|
+|MinIO|9000|
+
+As mentioned the virtual machine can be reached at `10.0.3.10`, meaning the services will be available at `10.0.3.10:<port-number>`. For convenience Consul, Vault, and Nomad have all been forwarded to `localhost` as well, meaning they are available at `localhost:<port-number>`. MinIO has _not_ been forwarded.  
+
+### CLI-tools
+Nomad, Vault and Consul have their own CLI-tools to interact with the servers that are running. They all default to `localhost`, and the default ports mentioned above. This means you can download any of the binaries, and they will automatically be connected to the services running inside the virtual machine. Refer to [this section](#iteration-of-the-development-process) to see examples on how to use this.
+
+### Using ansible
+When working with this box we will use a technology called [ansible](https://www.ansible.com/). In short, ansible is a software that logs onto a computer like a normal user, and performs tasks defined in an ansible playbook (example [template_example/dev/ansible/playbook.yml](template_example/dev/ansible/playbook.yml). We will mostly be using this to interact with our virtual machine. In our case _all_ playbooks put inside [dev/ansible/](./dev/ansible/) will be run every time we start the box, and we will utilise this throughout the guide.
+
+### Making artifacts available inside the box
+As mentioned, the local and virtual machine are separate. To make something from our local machine (for example our code) available to the virtual machine we've got two options: 
+
+#### 1. Synced folder
+As mentioned earlier the virtual machine and local machine have a shared folder. In _all_ vagrant boxes the folder in which the `Vagrantfile` lies will be linked to `/vagrant` inside the box. In our case the `Vagrantfile` is in our root directory, the same as where this `getting_started.md` is.
+
+#### 2. MinIO
+Anything put in MinIO will be available to the virtual machine. Two methods are available to upload to MinIO: 
+
+- 1. Upload files via the UI at `10.0.3.10:9000`. 
+- 2. _All_ files put in the root directory (same as this file) will be automatically synced with MinIO
+
+See [pushing docker image](#pushing-resources-to-minio-with-ansible-docker-image) and [fetching docker image](#fetching-resources-from-minio-with-nomad-docker-image) for examples on how to upload a docker image. We will be using this later in the guide.
```

Comment:
Is `pushing-resources-to-minio-with-ansible-docker-image` and `fetching-resources-from-minio-with-nomad-docker-image` upcoming sections?

Reply:
in README, forgot to change, will fix